pkg/releases/releases.go

```diff
@@ -133,6 +143,7 @@ var (
 	terraformReleaseRegex   = regexp.MustCompile(`- \[Terraform v(\d+\.\d+\.\d+)\] (https:\/\/registry\.terraform\.io\/providers\/(.*?)\/(.*?)\/.*?) - (.*)`)
 	rubyGemReleaseRegex     = regexp.MustCompile(`- \[Ruby Gems v(\d+\.\d+\.\d+)\] (https:\/\/rubygems\.org\/gems\/(.*?)\/versions\/.*?) - (.*)`)
 	nugetReleaseRegex       = regexp.MustCompile(`- \[NuGet v(\d+\.\d+\.\d+)\] (https:\/\/www\.nuget\.org\/packages\/(.*?)\/\d+\.\d+\.\d+) - (.*)`)
+	swiftReleaseRegex       = regexp.MustCompile(`- \[Swift Package Manager v(\d+\.\d+\.\d+)\] (https:\/\/(github.com\/.*?)\/releases\/tag\/.*?\/?v\d+\.\d+\.\d+) - (.*)`)
```

Comment:
```suggestion
	swiftReleaseRegex       = regexp.MustCompile(`- \[Swift v(\d+\.\d+\.\d+)\] (https:\/\/(github.com\/.*?)\/releases\/tag\/.*?\/?v\d+\.\d+\.\d+) - (.*)`)
```

Should this just say Swift since its not being published to a package manager?

Reply:
right. maybe i'm misunderstanding something here but my assumption was that we need to do something similar to go. changes added here: https://github.com/speakeasy-api/sdk-generation-action/pull/56/files#diff-dbb0536e4396142b02a85d24d4b73455e5c6ee90e11238be59830acefb847f2fR89

Reply:
I think it makes sense calling this 'Swift Package Manager' since the git release is what will be picked up and used by SPM